### DISCUSSION TITLE
Working on Windows 11 too.

### DISCUSSION THREAD

**Discussion by ghost** [extremecoders-re/go-dispatch-proxy#22] [Show and tell]
You should specifying the systems u tested or other users tests that confirms its working 100% your program on that systems.
I was used Win 10 and Win 11 and all work 100% no problems.

**Comment by wikiwyrhead** [comment]
windws 11 says unsupported SOCKS version

**Comment by Ubaaaa** [comment]
How to force win11 to use socks5? Got it to work with firefox, there i can specify to use sock5
